### PR TITLE
EOS-26561: removed hardcoding of secrets path from provisioner

### PR DESCRIPTION
**Pull request by lakshita-jain**
Signed-off-by: Lakshita Jain <lakshita.jain@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Review by kailashyadav11 — APPROVED** [review]
LGTM

File: src/provisioner/provisioner.py [hunk 1]
```diff
@@ -85,6 +85,10 @@ def config_apply(solution_config_url: str, cortx_conf_url: str = None,
         except ConfError as e:
             Log.error(f'Unable to load {solution_config_url} url, Error:{e}')
 
+        # Secrets path from config file
+        if cortx_conf.get('cortx>common>storage>local'):
+            CortxProvisioner.secrets_path =cortx_conf.get('cortx>common>storage>local')+'/solution/secret'
```
Comment (zahidsk) [inline src/provisioner/provisioner.py:90]: save relative path '/solution/secret' in a variable and then use it to form absolute path

Comment (nitin-seagate) [reply src/provisioner/provisioner.py:90]: Yes, please use constant define for relative path: `/solution/secret`

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

**Review by tanujashinde0405 — APPROVED** [review]
LGTM

**Review by nitin-seagate — APPROVED** [review]
LGTM

**Pull request merged**